### ISSUE TITLE
drag & drop media upload

### ISSUE THREAD

**Issue by MaybellEigenhart** [dimdenGD/OldTwitter#477]
Firefox, 1.7.7.3

My tweetdeck habits are dying hard and I keep dragging & dropping files onto the text field and losing my tweet, opening a new file upload dialogue every time I want to post an image is somewhat arduous. Possible/sensible feature to add?

**Comment by dimdenGD** [comment]
I don't understand what do you mean, you already can drag'n'drop media to tweet

**Comment by MaybellEigenhart** [comment]
Ah, I see what you mean, drag & drop functions for grabbing URLs from web media both in and out of browser (tested with external Discord client), however drag & drop does not allow for uploading files from File Explorer.

![image](https://github.com/dimdenGD/OldTwitter/assets/111300991/6f0af50c-4b4b-4b44-940a-c7ad539a2a87)

![image](https://github.com/dimdenGD/OldTwitter/assets/111300991/2398b4ea-dc0c-41f1-b543-7f8c14dc1c78)

![image](https://github.com/dimdenGD/OldTwitter/assets/111300991/87f8826c-36ed-41af-bc6a-09009591a195)

**Comment by dimdenGD** [comment]
Works fine for me
![gif](https://lune.dimden.dev/d0c37a60e6e.gif)

**Comment by MaybellEigenhart** [comment]
Very strange, is it possible this is a Firefox-exclusive bug? Or has this been tested on Firefox and is this as such an idiosyncrasy particular to my system?
![2zKMCN8qmi](https://github.com/dimdenGD/OldTwitter/assets/111300991/7bed818a-77e0-46eb-a88e-38beae93d28b)

**Comment by dimdenGD** [comment]
Yeah seems like it only doesn't work on Firefox, idk why, I tried few possible fixes but none worked
